src/components/LicenseRemindModal/index.jsx

```diff
@@ -78,38 +78,29 @@ class LicenseRemindModal extends React.Component {
     const {
       isBulkRemind,
       user,
+      subscriptionUUID,
+      searchQuery,
       sendLicenseReminder,
       fetchSubscriptionDetails,
       fetchSubscriptionUsers,
-      searchQuery,
     } = this.props;
     // Validate form data
     this.validateFormData(formData);
     // Configure the options to send to the assignment reminder API endpoint
     const options = {
-      template: formData['email-template-body'],
-      template_greeting: formData['email-template-greeting'],
-      template_closing: formData['email-template-closing'],
+      greeting: formData['email-template-greeting'],
+      closing: formData['email-template-closing'],
+      user_email: user.userEmail,
```

Comment:
Are we prematurely adding `user_email` to the `options` here? It's only relevant when `isBulkRemind` is false. See the below removed code email is only added to options if `isBulkRemind` is false. Is `user` defined when choosing the bulk remind option?

Reply:
Fixed.